addon/scheduler.ts

```diff
@@ -3,7 +3,7 @@ import { schedule } from '@ember/runloop';
 import { deprecate } from '@ember/debug';
 import Router from '@ember/routing/router';
 import { gte } from 'ember-compatibility-helpers';
-import { buildWaiter, Token } from 'ember-test-waiters';
+import { buildWaiter, Token } from '@ember/test-waiters';
```

Comment:
seem this can happen already?

Reply:
Ya, we just need to make sure we depend on at least `ember-test-waiters@2.1.0` or `@ember/test-waiters`.